### ISSUE TITLE
K8s deployment (via helm chart)

### ISSUE THREAD

**Issue by cniweb** [moyix/fauxpilot#224]
Is it possible to run the fauxpilot containers under a K8s cluster?
Is there a ready-to-use helm chart for this?

**Comment by jamiemoller** [comment]
I'm gonna have a stab at this soon if i get time

**Comment by fdegier** [comment]
Yes you can run it on k8s if you add an Nvidia GPU node, there is no Helm chart available.
